lib/puppet/provider/network_dns/ios.rb

```diff
@@ -21,6 +21,8 @@ def self.instances_from_cli(output)
       new_instance[:ensure] = 'present'
       new_instance[:search] = [].push(new_instance[:search]) if new_instance[:search].is_a?(String)
       new_instance[:servers] = [].push(new_instance[:servers]) if new_instance[:servers].is_a?(String)
+      # servers can come as either a single value or space separated list - deal with it (⌐■_■)
+      new_instance[:servers] = new_instance[:servers].flatten.map(&:split).flatten if new_instance[:servers]
```

Comment:
@tphoney @willmeek I don't love this bit - but could not figure out any magic regex to gather it correctly in a single pass.